YahooFantasy.mjs

```diff
@@ -66,16 +68,21 @@ class YahooFantasy {
       }
     };
 
-    request(options, (e, body, data) => {
-      if (e) {
-        return cb(e);
-      } else {
-        if (data.error) {
-          return cb(data.error);
+    return new Promise((resolve, reject) => {
+      request(options, (e, body, data) => {
+        if (e) {
+          reject(e);
+          return cb(e);
+        } else {
+          if (data.error) {
+            reject(data.error);
```

Comment:
any way we could combine these rejects?

```
const err = e || data.error;
if (err) {
  reject(err);
  return cb(err);
}
```

would that even work?

Reply:
I don't see why not